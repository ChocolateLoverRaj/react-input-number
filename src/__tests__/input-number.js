import React from 'react';
import renderer from 'react-test-renderer';
import InputNumber from '../input-number';

test('value', () => {
  expect(
    renderer.create(<InputNumber value={10} />).toJSON()
  ).toMatchSnapshot();
});
