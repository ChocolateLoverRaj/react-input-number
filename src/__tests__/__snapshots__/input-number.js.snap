// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`value 1`] = `
<input
  onChange={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  type="text"
  value={10}
/>
`;
